packages/ddp-server/livedata_server.js

```diff
@@ -1520,30 +1520,30 @@ Object.assign(Server.prototype, {
    * @summary Set publication strategy for the given publication. Publications strategies are available from `DDPServer.publicationStrategies`. You call this method from `Meteor.server`, like `Meteor.server.setPublicationStrategy()`
```

Comment:
Both JSDocs still suggest that it's set per publication, not collection.

Reply:
Thanks! I just changed them as well.